packages/hdwallet-keepkey/src/keepkey.ts

```diff
@@ -623,6 +623,7 @@ export class KeepKeyHDWallet implements core.HDWallet, core.BTCWallet, core.ETHW
   readonly _supportsDebugLink: boolean;
   readonly _isKeepKey = true;
   readonly _supportsETH = true;
+  readonly _supportsEthSwitchChain = true;
```

Comment:
is this really true for keepkey? @pastaghost @BitHighlander think this should be false right now